task-runner/task_runner/executers/seissol/executer.py

```diff
@@ -0,0 +1,33 @@
+import os
+import shutil
+
+from task_runner.executers import BaseExecuter, command
+
+
+class SeisSolExecuter(BaseExecuter):
+    def execute(self):
+        """
+        Execute the SeisSol simulation.
+
+        This method runs the SeisSol simulator using the specified input configuration
+        and stores the output in the artifacts directory.
+        """
+        input_dir = os.path.join(self.working_dir, self.args.sim_dir)
+        shutil.copytree(input_dir, self.artifacts_dir, dirs_exist_ok=True)
+
+        input_file_path = os.path.join(self.artifacts_dir, self.args.input_filename)
```

Comment:
You are not sending `input_filename` from the python client. And this simulator does not use it. I believe you can remove this line.